map2loop/mapdata.py

```diff
@@ -298,28 +295,66 @@ def get_colour_filename(self):
         return self.colour_filename
 
     @beartype.beartype
-    def set_ignore_codes(self, codes: list):
+    def set_ignore_lithology_codes(self, codes: list):
```

Comment:
We need to put this in the release notes that the set_ignore_codes has change to set_ignore_lithology_codes

Reply:
added to the todo before merge list in the PR above